api/.env.example

```diff
@@ -15,3 +15,4 @@ PORT=8080
 SESSION_SECRET={some secret}
 CG_SECRET={some secret}
 HEALTH_CHECK_SECRET={some secret}
+ALLOWED_REQUEST_ORIGIN=http://localhost:3000
```

Comment:
Since the demo deployment is in progress we need to make sure it's added to deployment env vars.